### PR TITLE
Make the whole Player card clickable

### LINKED ISSUE #343: [MyMeta] Make the whole Player card clickable
On My Meta homepage make the whole player card clickable so that it is easier to access the player profile page.
Current behavior: Player name is a link (clickable)
New behavior: Player Card is a link (clickable)

### PR DESCRIPTION
**Pull request by mquellhorst**
Tried to fix #343 
Works fine on Brave

### CODE REVIEW

**Review by dysbulic — CHANGES_REQUESTED** [review]
¿You just moved the link to the top of the tree, right?

It shows a ton of lines changed b/c of spacing, but it looks like everything is in the same places.

You do need to put the `.env.sample` back.

File: .env.sample [hunk 1]
```diff
@@ -1,15 +0,0 @@
-COMPOSE_PROJECT_NAME=the-game
```
[the remark this answers is not in the document]
Comment (mquellhorst) [reply .env.sample:1]: Yeah I did and thought I put it back before commiting.
Add the file the file again and recommit?

**Review by mquellhorst — COMMENTED** [review]
Correct and added text-decoration none so not all the text get underlined on hover

**Comment by dysbulic** [comment]
This overrides the Twitter, Github and Etherscan links in the footer of the card even though the correct link address shows at the base of the browser.

I don't know how to fix this. z-index, maybe? Probably not.

Also the text in the profile is now cyan and not white.

**Comment by dysbulic** [comment]
Apparently links within links is not a simple thing to address at all: https://stackoverflow.com/a/46707009

Unless you want to try to get this CSS-based example to work, this should probably go back to the issue, #343, for further discussion.

**Comment by mquellhorst** [comment]
I'll look into it whether I can fix this, thanks!

**Pull request closed without merging**